src/main/java/org/folio/bulkops/service/BulkOperationService.java

```diff
@@ -229,13 +228,10 @@ public void confirm(BulkOperation operation)  {
               isChangesPresented = true;
             }
             writerForModifiedJsonFile.write(modifiedRecord);
-          } else {
-            committedNumOfErrors++;
           }
+          operation.setCommittedNumOfErrors(operation.getCommittedNumOfErrors() + modified.getErrorsCount());
```

Comment:
At this line operation.getCommittedNumOfErrors() is already 0.

Reply:
it is necessary to increment it by errors

Reply:
This is DATA_MODIFICATION stage - method confirm(...) - so it is the very beginning of application. How can we have there committed errors?

Reply:
Hi @khandramai  , this errors value will be incremented by errors value form entity holder modified errors count value